packages/replayio/src/utils/recordings/upload/uploadRecording.ts

```diff
@@ -135,40 +136,51 @@ export async function uploadRecording(
 
   recording.uploadStatus = "uploaded";
 
-  if (processAfterUpload) {
-    debug("Begin processing recording %s ...", recording.id);
+  switch (processingBehavior) {
+    case "start-processing": {
+      debug("Start processing recording %s ...", recording.id);
 
-    try {
-      await client.waitUntilAuthenticated();
+      processRecording(client, { recordingId: recording.id }).catch(error => {
```

Comment:
it might be a good idea to add a comment here that we don't want to set processing statuses here as that would hurt the renderered results (they could get stuck in unfinished states etc as we are only waiting for uploads there)

Reply:
Yeah good call.